gtsfm/scene_optimizer.py

```diff
@@ -177,10 +177,11 @@ def create_computation_graph(
 
         # ensure cycle consistency in triplets
         cycle_consistent_graph = dask.delayed(cycle_consistency.filter_to_cycle_consistent_edges)(
-            i2Ri1_graph_dict, i2Ui1_graph_dict, two_view_reports_dict
+            i2Ri1_graph_dict, i2Ui1_graph_dict, v_corr_idxs_graph_dict, two_view_reports_dict
         )
         i2Ri1_graph_dict = cycle_consistent_graph[0]
         i2Ui1_graph_dict = cycle_consistent_graph[1]
+        v_corr_idxs_graph_dict = cycle_consistent_graph[2]
```

Comment:
you can avoid this and actually unpack the from the delayed object by specifying the nout= parameter.  Like: 
a, b, c = dask.delayed(myfunc, nout=3)(x, y)

Reply:
@akshay-krishnan good point, should be cleaned up now